data/mm/world/overworld.yml

```diff
@@ -90,12 +89,14 @@
     "Clock Town Fairy Fountain": "true"
     "Deku Playground": "true"
   events:
-    BOMBER_CODE: "has_weapon_range"
+    BOMBER_CODE: "has_weapon_range && (first_day || second_day || final_day)"
```

Comment:
Why? This will always be true

Reply:
Because you have to do it before it passes to the next day. i wasnt sure how to make sure it can do that other than this, if it even does.